presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -167,39 +167,37 @@ public void shouldRunQueryFromFile()
     private void launchPrestoCliWithServerArgument(String... arguments)
             throws IOException, InterruptedException
     {
-        if (!authentication) {
-            ImmutableList.Builder<String> prestoClientOptions = ImmutableList.builder();
-            prestoClientOptions.add(
-                    "--server", serverAddress,
-                    "--user", jdbcUser);
-            prestoClientOptions.add(arguments);
-            launchPrestoCli(prestoClientOptions.build());
+        ImmutableList.Builder<String> prestoClientOptions = ImmutableList.builder();
+        prestoClientOptions.add("--server", requireNonNull(serverAddress, "serverAddress is null"));
```

Comment:
Basically both messages are bad. But here you have the stack-trace at least. But i see you point. Just to keep it consistent i will change that.